### PR TITLE
Fix bundles with dir-shape have needed instruction in content.xml (#128)

### PR DESCRIPTION
**Pull request by doggy-dev**
When a bundles has the Eclipse-BundleShape: dir manifest entry the
installable unit in content.xml the touchpointData must have
<instruction key='zipped'>true</instruction>
Otherwise when tycho builds a product the bundle is not beeing
extracted.

### CODE REVIEW

**Comment by ctron** [comment]
Thanks for the PR. This looks good to me.

All that is requires is that you sign the Eclipse Contributor Agreement (ECA) and add the `Signed-off-by:` footer entry in the git commit, after you signed the agreement.

Let me know if you need any help with that.

File: bundles/org.eclipse.packagedrone.repo.aspect.common.tests/src/org/eclipse/packagedrone/repo/aspect/common/tests/InstallableUnitTest.java [hunk 1]
```diff
@@ -10,102 +10,226 @@
...
+        final boolean notOptional = false;
+        final Boolean nullGreedyBoolean = null;
+        final String noFilter = null;
+        final String bundleId = "b1";
+        final String feature2Id = "f2";
+        final String feature_id = "f1";
         final FeatureInformation fi = new FeatureInformation ();
-
-        fi.setId ( "f1" );
+        fi.setId ( feature_id );
```
Comment (ctron) [inline bundles/org.eclipse.packagedrone.repo.aspect.common.tests/src/org/eclipse/packagedrone/repo/aspect/common/tests/InstallableUnitTest.java]: Please don't use underscores. -> `featureId`.

Comment (ctron) [reply bundles/org.eclipse.packagedrone.repo.aspect.common.tests/src/org/eclipse/packagedrone/repo/aspect/common/tests/InstallableUnitTest.java]: Same for the other locations.

File: bundles/org.eclipse.packagedrone.repo.aspect.common.tests/src/org/eclipse/packagedrone/repo/aspect/common/tests/InstallableUnitTest.java [hunk 2]
```diff
@@ -10,102 +10,226 @@
...
 
-        fi.getRequirements ().add ( new Requirement ( Type.PLUGIN, "b1", Version.parseVersion ( "1.2.0" ), MatchRule.DEFAULT ) );
-        fi.getRequirements ().add ( new Requirement ( Type.PLUGIN, "b1", Version.parseVersion ( "1.3.0" ), MatchRule.DEFAULT ) );
+        final Touchpoint touchpoint = iu.getTouchpoints ().get ( 0 );
+        assertThat ( touchpoint.getInstructions (), hasEntry ( "zipped", is ( "true" ) ) );
+    }
 
-        final List<InstallableUnit> ius = InstallableUnit.fromFeature ( fi );
+    @Test
+    public void fragmentSecificPropertiesAddedToIU () throws Exception
```
Comment (ctron) [inline bundles/org.eclipse.packagedrone.repo.aspect.common.tests/src/org/eclipse/packagedrone/repo/aspect/common/tests/InstallableUnitTest.java]: Typo: "specific"

**Comment by doggy-dev** [comment]
I think I'm done. I also rebased the the commits on master.

**Comment by ctron** [comment]
Could you please rebase the branch. This should fix the issues with travis.

**Comment by doggy-dev** [comment]
Please don't merge yet. I think there is a bug in my code.

**Pull request merged**